src/utils/response.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable require-jsdoc */
+export default class Util {
+  constructor() {
+    this.statusCode = null;
+    this.type = null;
+    this.data = null;
+    this.message = null;
+  }
+
+  setSuccess(statusCode, message, data) {
+    this.statusCode = statusCode;
+    this.message = message;
+    this.data = data;
+    this.type = 'success';
+  }
+
+  setError(statusCode, message) {
+    this.statusCode = statusCode;
+    this.message = message;
+    this.type = 'error';
+  }
+
+  send(res) {
+    const result = {
+      status: this.type,
+      message: this.message,
+      data: this.data,
+    };
+
+    if (this.type === 'success') {
```

Comment:
Res type and status here was defined as string "status" or "error" but you return status as statusCode, please resolve.

Reply:
fixed